lib/live_isolated_component.ex

```diff
@@ -62,15 +68,28 @@ defmodule LiveIsolatedComponent do
       handle_event = socket |> store_agent_pid() |> StoreAgent.get_handle_event()
       original_assigns = socket.assigns
 
-      case handle_event.(event, params, normalize_socket(socket, original_assigns)) do
-        {:noreply, socket} ->
-          {:noreply, denormalize_socket(socket, original_assigns)}
+      result = handle_event.(event, params, normalize_socket(socket, original_assigns))
```

Comment:
Do we still need the `handle_event` and `handle_info` callback functions at all? I feel like they can be dropped.